pkg/lib/k8s/hpa.go

```diff
@@ -179,20 +190,35 @@ func IsHPAUpToDate(hpa *kautoscaling.HorizontalPodAutoscaler, minReplicas int32,
 		return false
 	}
 
-	if len(hpa.Spec.Metrics) != 1 {
+	if len(hpa.Spec.Metrics) != 2 {
+		return false
+	}
```

Comment:
Update this (and below in this function) if you make changes due to my comment above